src/RolloutScorer/RolloutScorerAzureFunction/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
```

Comment:
Why another gitignore? There isn't anything special about this project that should warrant another one different from the base.

Reply:
This was generated automatically; will remove.